src/views/Correction.vue

```diff
@@ -94,9 +94,9 @@ export default class Correction extends Mixins(CommonMixin, DateMixin, FilingTem
     return +this.$route.query['correction-id'] || 0
   }
 
-  /** The original filing datetime, in Pacific time. */
+  /** The original filing date, in Pacific time. */
   get originalFilingDate (): string {
-    return this.apiToPacificDateTime(this.getOriginalFilingDateTime)?.slice(0, 10)
+    return this.apiToPacificDateLong(this.getOriginalFilingDateTime)
```

Comment:
This is another bugfix.

Old:
![image](https://user-images.githubusercontent.com/10002889/166581026-468c3828-cda5-4468-9ea6-b70001a787e3.png)


New:
![image](https://user-images.githubusercontent.com/10002889/166581165-d654fcd4-049f-4295-9f3f-777f1819bddb.png)